ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1247,7 +1249,7 @@ export const RuntimePanel = fp.flow(
   // where we get 'status' from
   const runtimeCanBeUpdated = runtimeChanged
       && [RuntimeStatus.Running, RuntimeStatus.Stopped].includes(status as RuntimeStatus)
-      && runtimeCanBeCreated || pdSizeReduced;
+      && runtimeCanBeCreated || pdExists && pdSizeReduced;
```

Comment:
I'm pretty sure this is a bug without parens here. Even if it's not - this is unclear enough that we should add parens

Reply:
Do you mean by add parens to `(pdExists && pdSizeReduced)` ?